src/components/Attachments/AttachmentsActions.js

```diff
@@ -0,0 +1,49 @@
+import { createAction } from 'redux-actions';
+
+export const addUploadingFiles = createAction(
+  'ADD_UPLOADING_FILES',
+  (files, uploadFolderId) => ({
+    files,
+    uploadFolderId,
+  }),
+);
+
+export const fileUploadInitialized = createAction(
+  'FILE_UPLOAD_INITIALIZED',
+  (file, uploadFolderId) => ({ file, uploadFolderId }),
+);
+
+export const fileUploadGetLocation = createAction(
+  'FILE_UPLOAD_GET_LOCATION',
+  (file, uploadFolderId) => ({ file, uploadFolderId }),
+);
+
+export const fileUploadingStarted = createAction(
+  'FILE_UPLOADING_STARTED',
+  (file, uploadFolderId) => ({
+    file,
+    uploadFolderId,
+  }),
+);
+
+export const fileUploadingProgressUpdated = createAction(
+  'FILE_UPLOADING_PROGRESSS_UPDATED',
+  (file, uploadFolderId, progress) => ({ file, uploadFolderId, progress }),
+);
+
+export const fileUploadingSuccess = createAction(
+  'FILE_UPLOADING_SUCCESS',
+  (file, uploadFolderId, result) => ({ file, uploadFolderId, result }),
+);
+
+export const fileUploadingFailure = createAction(
+  'FILE_UPLOADING_FAILURE',
+  (file, uploadFolderId, error) => ({ file, uploadFolderId, error }),
+);
+
+export const retryFileUpload = createAction(
+  'RETRY_FILE_UPLOAD',
+  (file, uploadFolderId) => ({ file, uploadFolderId }),
+);
+
+export const clearList = createAction('CLEAR_LIST');
```

Comment:
`CLEAR_LIST` - which list, list of what?

Reply:
My bad, will fix it

btw, if the app structure will grow, we'll have a way to make modular redux actions in a next way:

![image](https://user-images.githubusercontent.com/20638114/73413730-3e7d6180-4315-11ea-9e97-dd22662ba1dc.png)

I didn't do it because we don't have agreements about redux actions, reducers, sagas and selectors structure. And this is not a necessary right now